src/background.js

```diff
@@ -69,6 +69,27 @@ async function main() {
         }
     }
 
+    async function bug(e) {
+        let [URL] =  await browser.tabs.query({active: true, currentWindow: true});
+        let domain = (URL.url).split("/")[2];
```

Comment:
It's never a good idea to parse URLs manually, and a convenient `URL` helper is available:
```
    let [tab] = await browser.tabs.query({active: true, currentWindow: true});
    let domain = new URL(tab.url).host;
```